src/app/cmdoptions.cpp

```diff
@@ -397,9 +416,11 @@ QStringList QBtCommandLineParameters::paramList() const
 QBtCommandLineParameters parseCommandLine(const QStringList &args)
 {
     QBtCommandLineParameters result {QProcessEnvironment::systemEnvironment()};
+    Q_ASSERT(args.size() > 0); // there has to be at least executable name
```

Comment:
Seems like wrong assertion. 
We call it with `QCoreApplication::arguments()` as argument. From its documentation:
>On Windows, the list is built from the argc and argv parameters only if modified argv/argc parameters are passed to the constructor. In that case, encoding problems might occur.
>
>Otherwise, the arguments() are constructed from the return value of GetCommandLine(). As a result of this, the string given by arguments().at(0) might not be the program name on Windows, depending on how the application was started.